src/Catalyst.Core.Lib.Tests/IntegrationTests/P2P/PeerRepositoryTests.cs

```diff
@@ -112,7 +115,7 @@ private void PeerRepo_Can_Update_And_Retrieve()
         }
         
         [Ignore("Setup to run in pipeline only")]
-        [Property(Traits.TestType, Traits.E2EMongoDb)]
+        [TestCategory(Traits.E2EMongoDb)]
```

Comment:
I see this is under the Microsoft.VisualStudio.TestPlatform.ObjectModel ref, while Category is under NUnit. Can this be Category instead of TestCategory or is TestCategory needed? I see in some places TestCategory is used and Category is used in others.